sdk/resourcemanager/Azure.ResourceManager/src/Resources/Custom/RestOperations/ResourcesRestOperations.cs

```diff
@@ -20,22 +20,22 @@ namespace Azure.ResourceManager.Resources
 {
```

Comment:
we can address in a future PR, but do we still need this restoperations in custom code?

Reply:
The main blocker is the `apiVersion` parameter for generic resource operations not correctly generated as a method parameter. I filed an issue to track it: https://github.com/Azure/autorest.csharp/issues/3094.